CHANGELOG.md

```diff
@@ -24,4 +24,5 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Module ``tt cat``, to print into stdout the contents of .snap/.xlog files.
 - Module ``tt play``, to play the contents of .snap/.xlog files to another Tarantool instance.
 - Module ``tt coredump``, to pack/unpack/inspect tarantool coredump.
-- Module ``tt run``, to start tarantool instance using tt wrapper. 
+- Module ``tt run``, to start tarantool instance using tt wrapper.
```

Comment:
AFAIS, the run module has nothing to do with this PR, please move this change to a separate commit for having clear history. [1]

Reply:
This is not a typo, this is missing newline symbol. Since, I have added new line, this change is related to the patch. I will not write a not on the same line.